tests/other/metadce/hello_world.jssize

```diff
@@ -1 +1 @@
-125595
+123041
```

Comment:
this is from the change to call a helper function rather than inline that code in each location?

Reply:
Yes, exactly